src/cluster/backend/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/zeromq/cmake")
+
+find_package(ZeroMQ)
+
+# Default to building ZeroMQ only if ZeroMQ was found.
+#
+# If a user enabled the cluster backend explicitly (-D ENABLE_CLUSTER_BACKEND_ZEROMQ:bool=ON),
```

Comment:
> I noticed that configure doesn't currently offer --disable-zeromq — did you consider this but opt to skip? I think it'd be good to have, since it's where people are accustomed to look and it's valid to want to build Zeek without it.

I'll go for `--disable-cluster-backend-zeromq`, as there might be other ZeroMQ based components. I did consider and mainly figured it won't be a big deal to build ZeroMQ if the libs are found (hard to use by mistake) and the `-D` toggle is good enough to disable.